### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by beyk**
I use media queries to make sure that my website displays correctly on desktop screens.

### CODE REVIEW

File: styles/desk-style.css [hunk 1]
```diff
@@ -0,0 +1,185 @@
+
+/* This applies from 600px onwards */
+@media (min-width: 992px) {
+  /* Header of the page */
+  .portfo-header {
+    margin-bottom: 40px;
+    margin-right: 50px;
+  }
+
```
Comment (rustam-tolipov) [inline styles/desk-style.css:9]: ## YOU DID A NICE JOB SO FAR 🎊👏

- [ ] However, there are some requirements that need to be fixed. If you follow all suggestion I gave you should be accepted by the next reviewer 💻🍀. In your hero section please add some more padding to the bottom to stick the Figma design 👌

#### Your design
image

#### Figma design, please ignore the yellow line. It means someone also making their portfolo. ok 😜?
image

File: styles/desk-style.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    height: 100%;
+    padding-top: 30px;
+    display: block;
+  }
+
+  .work {
+    margin-left: auto;
+    margin-right: auto;
+    width: 1156px;
+  }
```
Comment (rustam-tolipov) [inline styles/desk-style.css:71]: - [ ] There is too much space than it needs to be between the content of the work card. Please see the images below and decrease spacing in 3 places as I showed you 🤗

#### Your design
image

File: styles/desk-style.css [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+  .text-info ul li {
+    padding-right: 30px;
+  }
+
+  .about-me {
+    flex-direction: row;
+    min-height: 700px;
+  }
```
Comment (rustam-tolipov) [inline styles/desk-style.css:138]: - [ ] And finally, in the about section columns are not aligned in one line and there is a typo. Please make both sides in one line (marked with pink line) and of course don't forget to change `langguages` to `Languages` (marked with lime line). That's it, please fix all the mentioned issues, **good luck 🍀**

#### Screenshot the marks
image

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @beyk , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -5,243 +5,281 @@
...
+				<div class="hero-content">
+					<div class="hero-text">
+						<h1>I’m Beyk 
+							<br/>Glad to see you!</h1>
+						<p>
+							I’m a software developer!, I can help you build a product , feature or website Look through some of my work and
+							experience! If you like what you see and have a project you need coded, don’t hestiate
+							to contact me.
+						</p>
+						<a href="#">LET’S CONNECT</a>
```
Comment (brytebee) [inline index.html:35]: - [ ] **Required:** You are required to have a clone of the template as the previous reviewer has said. I observed the follow 

Your design | Figma template
:---: | :---:
image | image

All font-family should be Poppins. Let's make the change 👍 

- [ ] **Required:** Let's make the alignment similar as well on the card of projects and others, the language section should be aligned properly and the button in the contact section should be centred like all other elements in the section.

Your design | Figma template
:---: | :---:
image | image
image | image
image | image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @beyk👋, 

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by brytebee — APPROVED** [review]
Hi, @beyk 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**